pkg/ring/ring.proto

```diff
@@ -18,6 +18,7 @@ message IngesterDesc {
 message TokenDesc {
 	uint32 token = 1;
 	string ingester = 2;
+	string nextIngester = 3;
```

Comment:
Is this extra state redundant?  Could this be detected uniquely by a token being owned by a "PENDING" ingester?

Reply:
While the transfer is underway the tokens are still owned by the leaving ingester. The ingester recieving the chunks is in JOINING state.